multisig/proposals/ethereum/vip-049/index.ts

```diff
@@ -0,0 +1,83 @@
+import { parseUnits } from "ethers/lib/utils";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+const UPPER_BOUND_RATIO = parseUnits("1.01", 18);
+const LOWER_BOUND_RATIO = parseUnits("0.99", 18);
+const ezETH_ONE_JUMP_REDSTONE_ORACLE = "0x8062dC1b38c0b2CF6188dF605B19cFF3c4dc9b29";
+const ezETH_ONE_JUMP_CHAINLINK_ORACLE = "0xa87E10C6F6DAD7af6C17f82Ce2C00FA5C64d110c";
+export const ezETH = "0xbf5495Efe5DB9ce00f80364C8B423567e58d2110";
+export const vezETH = "0xA854D35664c658280fFf27B6eDC6C4195c3229B3";
+const BOUND_VALIDATOR = "0x1Cd5f336A1d28Dff445619CC63d3A0329B4d8a58";
+export const VTOKEN_RECEIVER = "0x1E3233E8d972cfFc7D0f83aFAE4354a0Db74e34E";
+export const INITIAL_SUPPLY = parseUnits("1.41", 18);
+export const SUPPLY_CAP = parseUnits("14000", 18);
+export const BORROW_CAP = parseUnits("1400", 18);
+const CHAINLINK_ezETH_FEED = "0x636A000262F6aA9e1F094ABF0aD8f645C44f641C";
+const REDSTONE_ezETH_FEED = "0xF4a3e183F59D2599ee3DF213ff78b1B3b1923696";
+const STALE_PERIOD_26H = 60 * 60 * 26; // 26 hours (pricefeeds with heartbeat of 24 hr) ;
+const CF = parseUnits("0.8", 18);
+const LT = parseUnits("0.85", 18);
+
+export const vip049 = () => {
+  return makeProposal([
+    // Configure Oracle
+    {
+      target: BOUND_VALIDATOR,
+      signature: "setValidateConfig((address,uint256,uint256))",
+      params: [[ezETH, UPPER_BOUND_RATIO, LOWER_BOUND_RATIO]],
+    },
+    {
+      target: ethereum.REDSTONE_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[ezETH, REDSTONE_ezETH_FEED, STALE_PERIOD_26H]],
```

Comment:
The heartbeat of ezETH/ETH on RedStone is 12 hours, not 24 hours. So, I would configure a max stale period of 13 hours. [RedStone docs](https://docs.redstone.finance/docs/get-started/price-feeds)

<img width="908" alt="Screenshot 2024-07-26 at 12 34 45" src="https://github.com/user-attachments/assets/7e891cbd-b4f5-4ff2-ae05-b72cdd10e45b">